core/controllers/release_coordinator.py

```diff
@@ -49,3 +54,121 @@ def get(self) -> None:
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'commit_message': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'new_rules': {
+                'schema': {
+                    'type': 'list',
+                    'items': {
+                        'type': 'object_dict',
+                        'object_class': parameter_domain.PlatformParameterRule
+                    }
+                },
+                'default_value': None
+            },
+        }
+    }
+
+    @acl_decorators.can_access_release_coordinator_page
+    def get(self) -> None:
+        """Handles GET requests."""
+        feature_flag_dicts = feature_services.get_all_feature_flag_dicts()
+        self.render_json({
+            'feature_flags': feature_flag_dicts,
+        })
+
+    @acl_decorators.can_access_release_coordinator_page
+    def post(self) -> None:
+        """Handles POST requests."""
+        assert self.normalized_payload is not None
+        action = self.normalized_payload.get('action')
+        try:
+            # The handler schema defines the possible values of 'action'.
+            # If 'action' has a value other than those defined in the
+            # schema, a Bad Request error will be thrown. Hence, 'action'
+            # must be 'update_feature_flag_rules' if this branch is
+            # executed.
+            assert action == 'update_feature_flag_rules'
+            feature_name = self.normalized_payload.get('feature_name')
+            if feature_name is None:
+                raise Exception(
+                    'The \'feature_name\' must be provided when the action'
+                    ' is update_feature_flag_rules.'
+                )
+            new_rules = self.normalized_payload.get('new_rules')
+            if new_rules is None:
+                raise Exception(
+                    'The \'new_rules\' must be provided when the action'
+                    ' is update_feature_flag_rules.'
+                )
+            commit_message = self.normalized_payload.get('commit_message')
+            if commit_message is None:
+                raise Exception(
+                    'The \'commit_message\' must be provided when the '
+                    'action is update_feature_flag_rules.'
+                )
+
+            assert self.user_id is not None
+            try:
+                feature_services.update_feature_flag_rules(
+                    feature_name, self.user_id, commit_message,
+                    new_rules)
+            except (
+                    utils.ValidationError,
+                    feature_services.FeatureFlagNotFoundException) as e:
+                raise self.InvalidInputException(e)
+
+            new_rule_dicts = [rules.to_dict() for rules in new_rules]
```

Comment:
rule.to_dict() for rule in new_rules

(unless each item in the list is multiple rules?)